jenkins/e2e-image/kubernetes-anywhere-e2e-runner.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Download the latest version of kubernetes-anywhere, then run the e2e tests using e2e-runner.sh.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+export KUBERNETES_ANYWHERE="${WORKSPACE}/kubernetes-anywhere"
+
+git clone https://github.com/kubernetes/kubernetes-anywhere.git "${KUBERNETES_ANYWHERE}"
```

Comment:
It looks like this file is trying to do two things:
1) Clone the kubernetes-anywhere repo, which is something bootstrap knows how to do already
2) Send options to `e2e-runner.sh`.

We can already do this without a new runner script and/or changing the kubekins-e2e image:
1) bootstrap already knows how to clone repos, which we use extensively in `bootstrap-repo.yaml`
2) We can setup the environment inside `ci-kubernetes-e2e-kubeadm-gce.sh`

Please add the ci-kubernetes-e2e-kubeadm-gce.sh job to bootstrap.yaml and configure it to clone the kubernetes-anywhere repo, and add the E2E_OPT value to the job-env section of the shell script.